packages/design-system/src/components/button/button.tsx

```diff
@@ -1,5 +1,5 @@
 import { clsx } from 'clsx';
-import { noop } from 'lodash';
+import { noop } from '@accelint/core';
```

Comment:
Just a question, why do we need a library function for a `noop`? I saw it in there and I was very confused.

Additionally, why would we take ownership of implementations in other libraries we use?

Reply:
I'm not suggesting a change at all, I am genuinely curious. From my perspective what I see is that with a `noop` as part of a library:

1. We have it to maintain - not that there is much to do for this to be clear
2. We have to import it to use it which is more laborious than just writing it

```
import { noop } from '@accelint/core'; // 38 characters
const noop = () => {}; // 22 characters
```

3. Additionally it is added footprint for the library:
    - Download + parse etc.
    - Dependency graph management

I just don't see the upside. I don't see much harm to be clear just raising it as a topic to discuss because I am new still.

Reply:
https://github.com/gohypergiant/standard-toolkit/discussions/83 for convo above

Reply:
I'll "pick" on point 2. Normal D.R.Y. things.  I don't want to remake it that many times (image). And the laborious is situational. Chances are, other items are also being used so then the count is just `5` (`6` if you count return)
![Screenshot 2024-11-05 at 9 51 50 AM](https://github.com/user-attachments/assets/bc855c1f-2777-4800-a08f-9e8e939b5226).